backend.config.js

```diff
@@ -1,21 +1,34 @@
+// Also wss://kusama-rpc.polkadot.io
+const DEFAULT_WS_PROVIDER_URL = "ws://kusama-rpc.polkadot.io";
+
 module.exports = {
-  // Local Polkadot Kusama node
-  wsProviderUrl: 'ws://substrate-node:9944',
-  // Postgres database connection params
+  wsProviderUrl: process.env.WS_PROVIDER_URL || DEFAULT_WS_PROVIDER_URL,
+
   postgresConnParams: {
-    user: 'polkastats',
-    host: 'postgres',
-    database: 'polkastats',
-    password: 'polkastats',
-    port: 5432,
+    user: "polkastats",
+    host: "postgres",
```

Comment:
Nooooo :crying_cat_face:

Reply:
Sorry about creating expectations

Reply:
It's fine. In the end victory will be ours! :muscle: